src/hockeypuck/server/cmd/cmd.go

```diff
@@ -5,13 +5,48 @@ import (
 	"os"
 	"path/filepath"
 	"runtime/pprof"
+	"strings"
+	"text/template"
 	"time"
 
 	"github.com/pkg/errors"
 
 	log "hockeypuck/logrus"
 )
 
+// Writer is a writer that writes to []byte
```

Comment:
Isn't this a reimplementation of bufio?

Reply:
It's quite similar, but not the full implementation.